pkg/kubelet/apis/cri/v1alpha1/runtime/api.proto

```diff
@@ -668,6 +668,7 @@ enum ContainerState {
     CONTAINER_RUNNING = 1;
     CONTAINER_EXITED  = 2;
     CONTAINER_UNKNOWN = 3;
+    CONTAINER_OOMKILLED = 4;
```

Comment:
I think oomkill should be a reason of exited state, not a standalone state. 

@yujuhong @Random-Liu WDYT?